templates/blocks/available-at-locations/available-at-locations.css

```diff
@@ -1,6 +1,9 @@
-.available-at-locations a {
-  font-weight: 400;
-  text-decoration: underline;
-  text-decoration-color: #afafaf;
-  color: var(--link-color);
-}
\ No newline at end of file
+.available-at-locations {
+  a {
```

Comment:
Just curious, is there a reason why this notation is preferred over .available-at-locations a?

Reply:
It's easier to read and maintaining css - https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting